.github/workflows/deploy.yml

```diff
@@ -0,0 +1,57 @@
+name: deploy
+
+on:
+  push:
+    branches: [dev, master, ci/deploy]
+    tags: ['v*.*.*']
+
+env:
+  AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_DEFAULT_REGION: 'eu-central-1'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Push Dev Image
+        if: github.ref == 'refs/heads/dev'
+        run: |
+          make build-and-push-docker
```

Comment:
Here you don't need to export the `DOCKER_IMAGE_TAG`?

Reply:
When it's not exported, the script creates a default for us with the `dev-` prefix